### ISSUE TITLE
WCAG issue: insufficient contrast

### ISSUE THREAD

**Issue by kristianeskogvang** [zazuko/Yasgui#42]
WCAG 1.4.3: The visual presentation of text and images of text has a contrast ratio of at least 4.5:1
https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html

Contrast between text and background is less than 4.5:1:
<img width="1055" height="465" alt="Image" src="https://github.com/user-attachments/assets/e0b490e8-0475-4f65-897e-180bb85fd0f8" />
